modules/Search/components/Facet.module.scss

```diff
@@ -1,29 +1,3 @@
-.container {
-    padding: $spacing-4;
-
-    &:not(:last-child) {
-        border-bottom: 1px solid $color-borders;
-    }
-}
-
-.header {
```

Comment:
_Comment placed in an unrelated place_

The hover color for search result title on dark theme makes the title disappear

https://user-images.githubusercontent.com/1641218/185591172-df390b36-9b1d-4211-af3f-4b4b39010388.mov